### ISSUE TITLE
LocalWriteStrobe output from eFPGA_Config

### ISSUE THREAD

**Issue by RobB720** [FPGA-Research-Manchester/FABulous#206]
Hi,

The LocalWriteStrobe output from eFPGA_Config doesn't seem to connect to any other blocks in my generated fabric and in the demo. Is this correct?

Best regards,
Rob.

**Comment by KelvinChung2000** [comment]
It is instantiated under the top wrapper. 

https://github.com/FPGA-Research-Manchester/FABulous/blob/eae6d7d6ad58c136857b99702c6275fb7bc6795e/fabric_generator/fabric_gen.py#L1837-L1863

**Comment by RobB720** [comment]
Yes it is an output from eFPGA_config instanciated in eFPGA_top, but it doesn't connect to anything, unless I am missing something?

Seems like a redundant signal, but want to make sure!

**Comment by KelvinChung2000** [comment]
I misread what you said and commented, which I found out later, so I deleted the comment. 

My answer is I am not sure and will need to ask @dirk-koch. But since we did not use it and could simulate without it, it is probably unnecessary?
